assessments/CogAT/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.csv
```

Comment:
@johncmerfeld One other minor thing - is this to avoid pushing student data to github? Couldn't this incidentally ignore changes to seeds if someone ran this? `git rm --cached *.csv`

Reply:
Yeah, I meant it as a safety mechanism (and so that my local `git status` would be clear about when I actually had unstaged changes), but it's definitely not perfect, given the legitimate CSVs used in the project. I'm happy to remove if you thyink it'll cause more problems than it solves

Reply:
Yeah that makes sense, I think maybe we remove for consistency across other bundles. But we may want to consider other safety mechanisms as a broader topic for this repo

Reply:
Makes sense -- removed!